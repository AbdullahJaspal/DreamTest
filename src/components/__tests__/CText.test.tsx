import 'react-native';
import React from 'react';
import CText from '../CText';

import {it} from '@jest/globals';
import renderer from 'react-test-renderer';

it('CText renders correctly', () => {
  // Render the CText component
  const tree = renderer.create(<CText>Test</CText>).toJSON();

  // Compare the rendered output to the saved snapshot
  expect(tree).toMatchSnapshot();
});
