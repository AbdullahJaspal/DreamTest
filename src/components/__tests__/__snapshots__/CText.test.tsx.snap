// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CText renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000000",
        "fontFamily": "BeVietNamPro-Regular",
        "fontSize": 14,
        "letterSpacing": 0.4,
        "lineHeight": 20,
      },
      {},
    ]
  }
>
  Test
</Text>
`;
